lib/round-tracker.js

```diff
@@ -1,6 +1,8 @@
 import Sentry from '@sentry/node'
 import { createMeridianContract } from './meridian/contract.js'
 
+export const TASKS_PER_ROUND = 30
```

Comment:
Is there a relationship between this variable and the round length?

Reply:
Sort of. We want to choose a number that will, on the one hand, allow most Stations to complete all tasks within one round and, on the other hand, keep Stations busy most of the time.

See https://www.notion.so/pl-strflt/SPARK-tasking-v2-604e26d57f6b4892946525bcb3a77104?pvs=4#ded1cd98c2664a2289453d38e2715643 for more details, this constant represents `TC` (tasks per committee).

We will need to tweak this value based on measurements; that's why I put it here as a constant.

I think this value will need to be much, much higher, like the low hundreds. However, we can't do that now because we have only 200 retrieval templates to sample from.

Reply:
got it, thank you! Could you add this context as a comment?

Reply:
Added a comment to explain - see [390249c](https://github.com/filecoin-station/spark-api/pull/80/commits/390249cb1ea33284568f427f704920ef3bd55be0)

Reply:
Thank you!